packages/imperative/src/rest/src/session/ConnectionPropsForSessCfg.ts

```diff
@@ -338,7 +360,7 @@ export class ConnectionPropsForSessCfg {
      * NOTE(Kelosky): redundant from LoggerUtils.SECURE_PROMPT_OPTIONS - leaving
      * for future date to consolidate
      */
-    private static secureSessCfgProps: Set<string> = new Set(["user", "password", "tokenValue", "passphrase"]);
+    public static secureSessCfgProps: Set<string> = new Set(["user", "password", "tokenValue", "passphrase"]);
```

Comment:
Quick question: Was this a required change to access to the set of secure props outside of this class?
I'm not opposed to doing this, but figured I would double-check, as I only see references to `this.secureSessCfgProps` in this PR (which would imply that we could leave it as `private` or `protected`).